CODE-OF-CONDUCT.md

```diff
@@ -34,6 +34,78 @@ In keeping with this commitment, we offer the following guidelines:
+- Discussions about these issues are important, and there are great avenues to
+  discuss them. This includes bringing up concerns to the [CNCF TOC][cncf-toc]
+  (which can be done through discussion with Security TAG leadership) or talking
+  to Security TAG leadership about moderating a post. To help ensure that we can
+  give focus to these issues and not tangle them up with technical discussions,
+  we should keep these discussions separate from channels which are focused
+  around technical exchange.
+
+For content creators:
+
+- Content must strive to remain _on-topic_, particularly where video and images
+  are provided.  Use of emojis and gifs as responses are content in and of
+  themselves need to be relevant to the particular post. For examples please
+  refer to the reference section below. 
+- If you receive a notice about a piece of content you've created, please seek
+  to understand that in some cases you may not agree with a decision or request.
+  Being able to practice tolerance and mindfulness is just as important to keep
+  the community working towards a common goal. The mediation and resolution
+  system that we have in place aims to handle this with the hope that both
+  content creators and consumers are heard and represented. These situations are
+  not zero sum, and often we aim to reach an agreeable compromise where a
+  discussion of a topic can happen without making members of the community feel
+  uncomfortable.
+- In the event where there is disagreement, we have some guidelines that can
+  help prevent escalation
+  - Do not bring the discussion out of context.
+  - Do not rationalize the actions you take. We do not expect anyone to
+    understand what everyone else feels towards certain things (e.g. the same
+    gestures in certain cultures are good and bad in others). Understand that
+    something may not be wrong, but it may affect others.
+
+In summary, be nice, inclusive and welcoming. Misunderstandings, mistakes and
+oversights happen, and when they do, there are some good ways to go about having
+a conversation with colleagues to make our community inclusive and welcoming to
+everyone!  
+
+## Reference
+
+Example of reasonable gif: Group is close to wrapping up deliverable, as part of
+an update, the lead posts a "nearly done" gif.  
+
+Example of reasonable emoji:  Post in the group uses emojis to break up content
+and is relevant to the item discussed or used in response to post to signify
+voting, opinion, acceptance, emotion, etc.  
+
+Example of reasonable image and video:  Posting a picture of a community meetup*
+or posting a recording to a presentation on cloud native security.
+
+*Note: Many events within the community may show alcohol consumption, it is
+important that when posting photos and videos members consider if the post
+glorifies alcohol or alcohol is the primary subject of the content
+(unacceptable) or if the alcohol is happenstance occurrence in the image
+(acceptable).  
+
```

Comment:
IMO this is very subjective. I mean who's to say if a video "glorifies alcohol" or not. So are we saying we're going to slap a warning label on the content or remove it altogether ?  Also wouldn't this need to be extended for drugs etc. I'm curious if any other org has similar guidelines.

Reply:
I agree Ashutosh, we need to extend it to number of things actually, including Drugs

Reply:
```suggestion
*Note: Many events within the community may include content which is only acceptable depending on the context it is used in.
An example of this is alcohol consumption. It is
important that when posting photos and videos members consider if the post
glorifies alcohol or alcohol is the primary subject of the content
(unacceptable) or if the alcohol is happenstance occurrence in the image
(acceptable).  

```

Changed it to be an example instead.

Reply:
A lot of this is very subjective.  We can provide some examples but realistically so long as everyone is not raised the same way this will continue to vary widely.  Example: drug use is not the same globally, some places items are classified as drugs and illegal or legal and in others they are not.